README.md

```diff
@@ -73,19 +73,22 @@ $ operator-sdk add api --api-version=app.example.com/v1alpha1 --kind=AppService
 # Add a new controller that watches for AppService
 $ operator-sdk add controller --api-version=app.example.com/v1alpha1 --kind=AppService
 
+# Set the username variable
+$ USERNAME=<username>
```

Comment:
```suggestion
$ export USERNAME=<username>
```

It is missing the export ^ @eranco74 could you please click on here for we are able to merge.